### PR TITLE
Add fields to update in update record action

### PR DESCRIPTION
**Pull request by thomtrp**
- update backend action so it handles composite fields
- add fields to update multiselect
- generate form based on that field
- add icons

### CODE REVIEW

File: packages/twenty-front/src/modules/workflow/workflow-actions/components/WorkflowEditActionFormUpdateRecord.tsx [hunk 1]
```diff
@@ -31,9 +36,23 @@ type WorkflowEditActionFormUpdateRecordProps = {
...
+  FieldMetadataType.Date,
+  FieldMetadataType.Boolean,
+  FieldMetadataType.Select,
+  FieldMetadataType.MultiSelect,
+  FieldMetadataType.Emails,
+  FieldMetadataType.Links,
+  FieldMetadataType.FullName,
+  FieldMetadataType.Address,
+];
+
```
Comment (Devessier) [inline packages/twenty-front/src/modules/workflow/workflow-actions/components/WorkflowEditActionFormUpdateRecord.tsx:55]: What is the reason behind this static list of field types?

**Review by Devessier — COMMENTED** [review]
It looks great! I have only one question about the static list of field types before we can merge.

File: packages/twenty-front/src/modules/workflow/workflow-actions/components/WorkflowEditActionFormUpdateRecord.tsx (hunk 1 above)
Comment (thomtrp) [reply packages/twenty-front/src/modules/workflow/workflow-actions/components/WorkflowEditActionFormUpdateRecord.tsx:55]: @Devessier I do not want the user to select a field a do not see the form to be updated. That would look weird. Some fields such as Rating may not be implemented before we get our first users in Beta. I prefer adding this list and remove it once we have all fields

Comment (Devessier) [reply packages/twenty-front/src/modules/workflow/workflow-actions/components/WorkflowEditActionFormUpdateRecord.tsx:55]: Makes sense.

**Review by Devessier — APPROVED** [review]
Another masterpiece by the pro developper @thomtrp. We can merge. Good job on the design of the multi select.

**Pull request merged**